### PR TITLE
8.0 account analytic project id new api

### PR DESCRIPTION
**Pull request by lmignon**
- use new api for account_analytic_project_id
- replace init method by a post_init hook
- move module description to README.rst

### CODE REVIEW

File: account_analytic_project_id/__openerp__.py [hunk 1]
```diff
@@ -30,29 +30,17 @@
...
-===========================
-
-This module adds a project_id field on analytic account so
-it becomes possible to write record rules restricting access
-to analytic accounts based on project properties (such as members).
-""",
-    "data": [],
-    "demo": [],
-    "test": [],
     "active": False,
```
Comment (Olivier-LAURENT) [inline account_analytic_project_id/__openerp__.py]: this line can be removed

File: account_analytic_project_id/account_analytic_project_id.py [hunk 2]
```diff
@@ -28,47 +28,23 @@
...
-class account_analytic_account(osv.Model):
+class account_analytic_account(models.Model):
     _inherit = 'account.analytic.account'
 
-    _columns = {
-        'project_id': fields.many2one('project.project', 'Project',
-                                      ondelete='set null'),
-    }
+    project_id = fields.Many2one(
+        'project.project', 'Project', ondelete='set null', copy=False,
```
Comment (Olivier-LAURENT) [inline account_analytic_project_id/account_analytic_project_id.py]: `set null` is the default value for the property `ondelete`

**Comment by Olivier-LAURENT** [comment]
2 minor details :+1:

File: account_analytic_project_id/README.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+===========================
+
+This module adds a project_id field on analytic account so
+it becomes possible to write record rules restricting access
+to analytic accounts based on project properties (such as members).
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/acsone/issues>`_.
```
Comment (adrienpeiffer) [inline account_analytic_project_id/README.rst]: `OCA/acsone` ?

File: account_analytic_project_id/README.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
```
Comment (adrienpeiffer) [inline account_analytic_project_id/README.rst]: Can you add  AGPL-3 ?

**Comment by lmignon** [comment]
@adrienpeiffer Done

**Comment by adrienpeiffer** [comment]
:+1: Thanks !

**Comment by Olivier-LAURENT** [comment]
:+1:

**Comment by sbidoul** [comment]
:+1:

**Pull request merged**